launch/tier4_planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.xml

```diff
@@ -80,7 +80,6 @@
           <!-- params -->
           <param name="input_topic" value="$(var interface_input_topic)"/>
           <param name="output_topic" value="path_smoother/path"/>
-          <param name="type" value="autoware_planning_msgs/msg/Trajectory"/>
```

Comment:
memo:

topic_realy can automatically detemin message type, it does not have `type` param

https://github.com/ros-tooling/topic_tools/blob/humble/topic_tools/src/relay_node.cpp